### PR TITLE
chore: lock react spring version

### LINKED ISSUE #5988: 依赖的@react-spring报错，看了一下是昨天发了一个新版本@9.7.0 导致打包失败
### Version of antd-mobile

latest

### Operating system and its version

_No response_

### Browser and its version

_No response_

### Sandbox to reproduce

_No response_

### What happened?

image

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by zombieJ**
resolve #5988

### CODE REVIEW

**Review by afc163 — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
     "@commitlint/config-conventional": "^17.3.0",
     "@docsearch/react": "^3.3.0",
     "@jest/types": "^28.1.3",
-    "@react-spring/core": "^9.5.5",
+    "@react-spring/core": "~9.6.0",
```
Comment (afc163) [inline package.json:49]: deps 排序也有点问题。这两离这么远

Comment (zombieJ) [reply package.json:49]: 一个是 dep 一个是 devDeps

**Review by afc163 — APPROVED** [review]

**Review by MadCcc — APPROVED** [review]

**Comment by xiaoshui222** [comment]
项目中怎么改呢

**Comment by ytianzaishui** [comment]
ERROR in ./node_modules/@react-spring/core/node_modules/@react-spring/animated/dist/index.mjs 177:38-39
    Can't import the named export 'hasFluidValue' from non EcmaScript module (only default export is available)
     @ ./node_modules/@react-spring/core/dist/index.js
     @ ./node_modules/@react-spring/web/dist/index.mjs
     @ ./node_modules/antd-mobile-v5/es/utils/reduce-and-restore-motion.js
     @ ./node_modules/antd-mobile-v5/es/index.js
     @ ./src/pages/uploadPhoto/uploadWithExCard/index.tsx
     @ ./src/.umi-production/core/routes.ts
     @ ./src/.umi-production/umi.ts
     @ multi ./src/.umi-production/umi.ts
    
    ERROR in ./node_modules/@react-spring/core/node_modules/@react-spring/animated/dist/index.mjs 205:24-25
    Can't import the named export 'hasFluidValue' from non EcmaScript module (only default export is available)
     @ ./node_modules/@react-spring/core/dist/index.js
     @ ./node_modules/@react-spring/web/dist/index.mjs
     @ ./node_modules/antd-mobile-v5/es/utils/reduce-and-restore-motion.js
     @ ./node_modules/antd-mobile-v5/es/index.js
     @ ./src/pages/uploadPhoto/uploadWithExCard/index.tsx
     @ ./src/.umi-production/core/routes.ts
     @ ./src/.umi-production/umi.ts
     @ multi ./src/.umi-production/umi.ts
    
    ERROR in ./node_modules/@react-spring/web/dist/index.mjs 113:9-10
    Can't import the named export 'is' from non EcmaScript module (only default export is available)
     @ ./node_modules/antd-mobile-v5/es/utils/reduce-and-restore-motion.js
     @ ./node_modules/antd-mobile-v5/es/index.js
     @ ./src/pages/uploadPhoto/uploadWithExCard/index.tsx
     @ ./src/.umi-production/core/routes.ts
     @ ./src/.umi-production/umi.ts
     @ multi ./src/.umi-production/umi.ts
    
    ERROR in ./node_modules/@react-spring/web/dist/index.mjs 116:9-10
    Can't import the named export 'is' from non EcmaScript module (only default export is available)
     @ ./node_modules/antd-mobile-v5/es/utils/reduce-and-restore-motion.js
     @ ./node_modules/antd-mobile-v5/es/index.js
     @ ./src/pages/uploadPhoto/uploadWithExCard/index.tsx
     @ ./src/.umi-production/core/routes.ts
     @ ./src/.umi-production/umi.ts
     @ multi ./src/.umi-production/umi.ts
    
    ERROR in ./node_modules/@react-spring/web/dist/index.mjs 118:7-8
    Can't import the named export 'is' from non EcmaScript module (only default export is available)
     @ ./node_modules/antd-mobile-v5/es/utils/reduce-and-restore-motion.js
     @ ./node_modules/antd-mobile-v5/es/index.js
     @ ./src/pages/uploadPhoto/uploadWithExCard/index.tsx
     @ ./src/.umi-production/core/routes.ts
     @ ./src/.umi-production/umi.ts
     @ multi ./src/.umi-production/umi.ts
    
    ERROR in ./node_modules/@react-spring/web/dist/index.mjs 145:25-26
    Can't import the named export 'is' from non EcmaScript module (only default export is available)
     @ ./node_modules/antd-mobile-v5/es/utils/reduce-and-restore-motion.js
     @ ./node_modules/antd-mobile-v5/es/index.js
     @ ./src/pages/uploadPhoto/uploadWithExCard/index.tsx
     @ ./src/.umi-production/core/routes.ts
     @ ./src/.umi-production/umi.ts
     @ multi ./src/.umi-production/umi.ts

**Pull request merged**